tests/5.0/master_taskloop/test_master_taskloop.F90

```diff
@@ -0,0 +1,61 @@
+!/===--- test_master_taskloop.F90 -------------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test checks the master taskloop directive in a parallel region. The
+! test performs simple operations on an int array which are then checked for
+! correctness.
+!
+!/===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_master_taskloop
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
```

Comment:
Runs with GCC 12 (no offloading device on the laptop):
`[OMPVV_RESULT test_master_taskloop.F90] Test passed on the device.`

```suggestion

OMPVV_TEST_OFFLOADING
```